app/src/main/java/agersant/polaris/PolarisPlaybackService.java

```diff
@@ -116,15 +159,59 @@ public void onReceive(Context context, Intent intent) {
 		autoSaveHandler.postDelayed(autoSaveRunnable, 5000);
 
 		pushSystemNotification();
+
+		mediaSession = new MediaSessionCompat(this, getPackageName());
+		mediaSession.setCallback(new Callback(player));
+        mediaSession.setActive(true);
+
+		updateMediaSessionState(PlaybackStateCompat.STATE_NONE);
+
+		dataUpdateRunnable = () -> {
+			updateMediaSessionState(player.isPlaying() ? PlaybackStateCompat.STATE_PLAYING : PlaybackStateCompat.STATE_PAUSED);
+			dataUpdateHandler.postDelayed(dataUpdateRunnable, 5000);
```

Comment:
Can you add a constant at the top of the file for the update period mentioned here and line 208?